### DISCUSSION TITLE
Easy Way to Refactor Notes-Parsing a MOC

### DISCUSSION THREAD

**Discussion by ShreyaPatibanda** [BryanJenksCommunity/FAQ#193] [Q&A] [answered]
At 1:43:10 of the  2021 Comprehensive Obsidian Zettelkasten Workflow, you mention when lots of notes point to a moc (ex: psychology), you may do some parsing (divide moc psychology to psychology concepts and notes connect to those). 

How do you do this? How do you easily change a mass number of notes to point to various psychology concepts? Is there an easy way to do this or you manually go through every note pointed to psychology and change it to a concept?

Thanks in Advance!

**Comment by tallguyjenks** [comment, marked as answer]
it would be a manual process, the thought is that going through everything manually and updating them is just another chance to lay eyes on all the related notes which is just another opportunity to inspire random connections
